### PR TITLE
[Ready for review] Updates to Boost your Research Reproducibility workshop folder for JOSE submission

### LINKED ISSUE #536: Prepare Boost your Research Reproducibility folder for submission to JOSE
### Summary

Tidy up and add documentation to the [Boost your Research Reproducibility with Binder folder](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/boost-research-reproducibility-binder) prior to submission to [JOSE](https://jose.theoj.org/about).

### What needs to be done?

- [x] Add HackMD notes to README
- [x] Add presenter/facilitators guide
- [ ] Write report summarising Manchester and London workshops and feedback
- [x] Add a short Markdown file titled paper.md with title, author names and affiliations, containing a description of the software, statement of need, and key references.

### Who can help?

* @rainsworth and @rosiehigman working on this 23/05
* Anyone who helped at or attended one of the workshops!

---

### Updates

**[What to include in the paper](https://jose.theoj.org/about#author_guidelines)**

JOSE papers should:
- [x] List all authors and affiliations.
- [ ] Describe the submission, and explain its eligibility for JOSE.
- [x] Include a _Statement of Need_, explaining how the submitted artifacts contribute to computationally enabled teaching and learning, and describing how they might be adopted by others.
- [x] For learning modules, describe the learning objectives, content, instructional design, and experience of use in teaching and learning situations.
- [x] Tell us the "story" of the project: how did it come to be?
- [x] List key references including a link to the open archive of the software or the learning module.

### PR DESCRIPTION
**Pull request by rosiehigman**
### Summary

Does not close #536 --- This PR was merged by @KirstieJane because she needed it to run a workshop. There is lots of amazing work here....but it doesn't quite address the submission requirements for JOSE. A new PR needs to be re-opened with the checklist for us to work thought.

This pull request includes additional files for the Boost your research reproducibility with Binder workshops and draft of the paper to submit to the [Journal of Open Source Education](https://jose.theoj.org/).

### List of changes proposed in this PR (pull-request)

* README file updated with HackMD notes and links to additional files
* paper.md with title, author names and affiliations, containing a description of the software, statement of need, and key references (in paper.bib) for JOSE submission
* instructor_guide.md with full instructions for running the workshop
* report.md with summary of feedback

### What should a reviewer concentrate their feedback on?

- [ ] Typos, language, check that it makes sense
- [ ] Everything looks ok?
- [ ] JOSE papers should:
  - [ ] List all authors and affiliations.
  - [ ] Describe the submission, and explain its eligibility for JOSE.
  - [ ] Include a _Statement of Need_, explaining how the submitted artifacts contribute to computationally enabled teaching and learning, and describing how they might be adopted by others.
  - [ ] For software submissions, describe the functionality of the software, usage and recent experience of use in teaching and learning situations.
  - [ ] For learning modules, describe the learning objectives, content, instructional design, and experience of use in teaching and learning situations.
  - [ ] Tell us the "story" of the project: how did it come to be?
  - [ ] List key references including a link to the open archive of the sofware or the learning module.

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by rainsworth** [comment]
Hi @r-j-arnold, @LouiseABowler, @sgibson91, @pherterich, @rosiehigman, @annakrystalli, @alexmorley, @martintoreilly, @ajstewartlang & @KirstieJane - this is ready for a first round of review.

I've added some of the [JOSE author guidelines](https://jose.theoj.org/about#author_guidelines) to the pull request comment to take into account when reviewing.

**Comment by sgibson91** [comment]
@rainsworth @rosiehigman If you merge `alan-turing-institute/master` into this branch, CI should pass now.

File: workshops/boost-research-reproducibility-binder/paper.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+# Statement of Need
+
+Reproducible research is necessary to ensure that scientific work can be trusted. 
+Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. 
+The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. 
+Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*[The Turing Way](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)* is a project [funded](https://www.turing.ac.uk/news/alan-turing-institute-spearhead-new-cutting-edge-data-science-and-artificial-intelligence) as part of UKRI’s Strategic Priorities fund.
```
Comment (pherterich) [inline workshops/boost-research-reproducibility-binder/paper.md]: The official acknowledgment @KirstieJane has added to the zenodo handbook record is "This work was supported by The UKRI Strategic Priorities Fund under the EPSRC Grant EP/T001569/1, particularly the "Tools, Practices and Systems" theme within that grant, and by The Alan Turing Institute under the EPSRC grant EP/N510129/1."

File: workshops/boost-research-reproducibility-binder/paper.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+Further chapters in development include coding style for reproducibility, credit for reproducible research and reproducible data analysis pipelines for machine learning, and will build upon Alan Turing Institute case studies. 
+The project demonstrates open and transparent project management and communication with future users, as it is openly developed at the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way). 
+It also facilitates workshops to train researchers and research software engineers to use tools to make their work more reproducible - one of which is the *Boost your research reproducibility with Binder* workshop.
+
+The Binder project significantly lowers the bar to sharing and re-using software, as it allows a user to try out someone else's work simply by clicking a single link. 
+The majority of researchers and educators have not yet discovered the Binder project, or they have heard it mentioned but do not know how to get started [@build_a_binder].
```
Comment (pherterich) [inline workshops/boost-research-reproducibility-binder/paper.md]: Is that researchers in general or the attendees of our workshop? Is "60% of all researchers know about Binder" a realistic goal?

File: workshops/boost-research-reproducibility-binder/paper.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Further chapters in development include coding style for reproducibility, credit for reproducible research and reproducible data analysis pipelines for machine learning, and will build upon Alan Turing Institute case studies. 
+The project demonstrates open and transparent project management and communication with future users, as it is openly developed at the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way). 
+It also facilitates workshops to train researchers and research software engineers to use tools to make their work more reproducible - one of which is the *Boost your research reproducibility with Binder* workshop.
+
+The Binder project significantly lowers the bar to sharing and re-using software, as it allows a user to try out someone else's work simply by clicking a single link. 
+The majority of researchers and educators have not yet discovered the Binder project, or they have heard it mentioned but do not know how to get started [@build_a_binder]. 
+It is also a tool that requires some investment to master.
```
Comment (pherterich) [inline workshops/boost-research-reproducibility-binder/paper.md]: Hmm, if you know what you're doing, setting up a Binder takes 10 minutes. Could we reword this to something around "the tool is most impactful if some time is invested to understand its potential and workings" or something similar?

File: workshops/boost-research-reproducibility-binder/paper.md (hunk 2 above)
Comment (rainsworth) [reply workshops/boost-research-reproducibility-binder/paper.md]: @pherterich in general? reference here: https://build-a-binder.github.io/

I can reword to "many researchers..."

File: workshops/boost-research-reproducibility-binder/paper.md (hunk 1 above)
Comment (rainsworth) [reply workshops/boost-research-reproducibility-binder/paper.md]: 👍 I've added an acknowledgement section.

File: workshops/boost-research-reproducibility-binder/paper.md (hunk 3 above)
Comment (rainsworth) [reply workshops/boost-research-reproducibility-binder/paper.md]: 👍

File: workshops/boost-research-reproducibility-binder/paper.md (hunk 2 above)
Comment (pherterich) [reply workshops/boost-research-reproducibility-binder/paper.md]: "The majority of researchers and educators interested in reproducible research" maybe?

Comment (rainsworth) [reply workshops/boost-research-reproducibility-binder/paper.md]: 👍

**Comment by rainsworth** [comment]
Ok, I am now going to merge `master` into this branch to fix CI.

**Comment by rainsworth** [comment]
I think the only thing left to do is add some tweets to the feedback report, so I will do that next.

File: workshops/boost-research-reproducibility-binder/paper.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+The [handbook](https://the-turing-way.netlify.com) currently includes chapters and training material on reproducibility, open research, version control, collaborating on GitHub/GitLab, research data management, reproducible environments, analysis testing, reviewing, continuous integration, reproducible research with Make and risk assessment. 
+Further chapters in development include coding style for reproducibility, credit for reproducible research and reproducible data analysis pipelines for machine learning, and will build upon Alan Turing Institute case studies. 
+The project demonstrates open and transparent project management and communication with future users, as it is openly developed at the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way). 
+It also facilitates workshops to train researchers and research software engineers to use tools to make their work more reproducible - one of which is the *Boost your research reproducibility with Binder* workshop.
+
+The Binder project significantly lowers the bar to sharing and re-using software, as it allows a user to try out someone else's work simply by clicking a single link.
```
Comment (annakrystalli) [inline workshops/boost-research-reproducibility-binder/paper.md]: Am just wondering whether sharing and reusing is the leading benefit of binder? It's my impression that github, good documentation and dependency management are the main drivers of sharing and reusability. To me, binder supports the ability to instantly reproduce and interact with shared code. Admittedly this might just be a comment on semantics. I generally use the terms as described here: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758530/

**Comment by annakrystalli** [comment]
Sorry for late input. All in all, it looks awesome to me! I just added I minor comment

File: workshops/boost-research-reproducibility-binder/paper.md (hunk 4 above)
Comment (rainsworth) [reply workshops/boost-research-reproducibility-binder/paper.md]: Thanks @annakrystalli, I've added that in!

**Comment by KirstieJane** [comment]
Merging because I'm about to run a workshop!
